gateway/README.md

```diff
@@ -0,0 +1,39 @@
+# javascript.client
```

Comment:
That's what I meant, but I see that I mistook this for a parent README. Consider my original comment resolved.